kratos/geometries/line_2d.h

```diff
@@ -481,7 +481,17 @@ class Line2D : public Geometry<TPointType>
     */
     virtual Vector& DeterminantOfJacobian(Vector& rResult, IntegrationMethod ThisMethod) const
     {
-        KRATOS_ERROR << "Jacobian is not square" <<  std::endl;
+        const unsigned int integration_points_number = msGeometryData.IntegrationPointsNumber( ThisMethod );
```

Comment:
Here the determinant of the Jacobian is computed. Such magnitude is constant for the 2-nodes line and the 3-nodes triangle.